docs/docusaurus/docs/reference/learn/gx_in_your_data_pipeline/delivery.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_label: 'Delivery'
+title: 'Delivery'
+description: Learn how GX can alert you to shifts in data metrics before it reaches BI tools.
```

Comment:
There's some number disagreement in this sentence and I think it's unclear what the pronoun "it" refers to. 


```suggestion
description: Learn how GX can alert you to shifts in data metrics before trusted data reaches BI tools.
```